### DISCUSSION TITLE
Auto accept party

### DISCUSSION THREAD

**Discussion by vo1ced** [isid0re/SoloLeveling#354] [General]
Hello, again! I'm making progress. Well, your bot is, kind of. Thank you!

Now that it's been settled one shouldn't fiddle with giving the leveling char items, I was wondering if it's a possibility to auto-accept party if I join the game with my char (to give BO or chant for example.) Auto-accept would be safe (as I'm playing HC) since only I know the pw to the game.

Thank you again, this is really cool and I can't wait to see how it will go on higher levels.

**Comment by isid0re** [comment]
the bot doesn't have any commands integrated into SoloLeveling for party activities such as recieving chants or BO.  You will have to do so manually in the solo game if you wantto do so.
